domains/hroot.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Hans Root",
```

username doesnt match github username
```suggestion
        "username": "H-Root",
```